src/modules/scraper/entry-point/http/controller.ts

```diff
@@ -20,6 +25,17 @@ export class ScraperController {
     });
   }
 
+  @Post("scraper/prices")
+  @ApiTags("scraper")
+  async submitPricesJobs(@Body() body: ProcessPricesBody) {
+    const { fromDepositId, toDepositId } = body;
+    for (let depositId = fromDepositId; depositId <= toDepositId; depositId++) {
+      await this.scraperQueuesService.publishMessage<TokenPriceQueueMessage>(ScraperQueue.TokenPrice, {
+        depositId,
+      });
+    }
+  }
```

Comment:
This is a new endpoint that allows to submit custom messages to the queue that's responsible for getting the token price. This is how we'll update the DB data without re-scraping everything

Reply:
AH very nice